### PR TITLE
22534-split-Refactoring2-package-to-extract-Tests

### PR DESCRIPTION
**Pull request by estebanlm**
https://pharo.manuscript.com/f/cases/22534/split-Refactoring2-package-to-extract-Tests

### CODE REVIEW

File: src/BaselineOfIDE/BaselineOfIDE.class.st [hunk 1]
```diff
@@ -365,6 +365,7 @@ BaselineOfIDE >> baseline: spec [
 
 		spec package: 'Refactoring2-Core'.
 		spec package: 'Refactoring2-Transformations'.
+		spec package: 'Refactoring2-Transformations-Tests'.
```
Comment (jecisc) [inline src/BaselineOfIDE/BaselineOfIDE.class.st:368]: I think it also needs to be in the groups.

**Review by jecisc — CHANGES_REQUESTED** [review]

**Comment by pavel-krivanek** [comment]
and failing SystemDependenciesTest

**Comment by estebanlm** [comment]
why? it is related?

**Comment by estebanlm** [comment]
so? @pavel-krivanek can I integrate this?

**Comment by pavel-krivanek** [comment]
yes

**Pull request merged**